hmda-regulator/src/main/resources/application.conf

```diff
@@ -0,0 +1,49 @@
+akka {
+  log-level = INFO
+  http.server.preview.enable-http2 = on
+  http.server.request-timeout=20800s
+  //Dev Schedules (fire every 5 minutes)
+  PanelScheduler = """akka.quartz.schedules.PanelScheduler.expression = "0 0/5 * * * ?""""
+  LarScheduler = """akka.quartz.schedules.LARScheduler.expression = "0 0/6 * * * ?""""
+  TsScheduler = """akka.quartz.schedules.TsScheduler.expression = "0 0/7 * * * ?""""
+  //Prod Schedules
+  //fire at 11:00PM everyday
+ // PanelSchedule = """akka.quartz.schedules.PanelSchedule.expression = "00 00 22 ? * *"""
+  //fire at 11:10PM everyday
+ // LARSchedule = """akka.quartz.schedules.LARSchedule.expression ="00 10 22 ? * *"""
+  //fire at 11:20PM everyday
+ // TSchedule = """akka.quartz.schedules.TSchedule.expression = "00 20 22 ? * *"""
+}
+
+
+hmda {
+  regulator {
+    http {
+      createSchema = true
+      createSchema = ${?INSTITUTION_PG_CREATE_SCHEMA}
+      timeout = 3
+      host = ${?HTTP_REGULATOR_HOST}
```

Comment:
I think it would be easiest if you put a default for the host and port the same way you did for grpc.